includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
```

Comment:
@adam-vessey Also, would this suggested change be done INSTEAD OF throwing an exception? Seems like this is still trying to go the route of logging nonexistent objects and avoiding exceptions. I'll wait for others to weigh in on this.